tests/test_remote_door.py

```diff
@@ -153,6 +154,8 @@ def test_get_remote_object(self):
         self.session = mock.MagicMock(name='session')
         self.session.client = "ssh"
         remote_door.Pyro4 = mock.MagicMock()
+        remote_door.server = mock.MagicMock()
+        remote_door.nameserver = mock.MagicMock()
```

Comment:
Since those are suppose to be created on import I think this setup should be cleared between tests. How about moving it to `setUp/tearDown` to ensure it's always as expected? (running a subset of tests or in different order might result in `remote_door` being different)

Reply:
The problem is that the imports might not always be available, after all isolation tests can be run in complete isolation. This then means that there is no easy way to just patch these and recover the original import (which might not exist) afterwards. But I will look for some way as you have a point here.